package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "commercetools-sunrise-data",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "config": {
+    "authUrl": "https://auth.commercetools.com",
+    "apiUrl": "https://api.commercetools.com"
+  },
+  "scripts": {
+    "set:projectKey": "npm config set commercetools-sunrise-data:project",
+    "set:clientId": "npm config set commercetools-sunrise-data:client_id",
+    "set:clientSecret": "npm config set commercetools-sunrise-data:client_secret",
+    "get:config": "echo \"projectKey\": && npm config get commercetools-sunrise-data:project && echo \"clientId\": && npm config get commercetools-sunrise-data:client_id && echo \"clientSecret\": && npm config get commercetools-sunrise-data:client_secret",
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
```

Comment:
build seems to be triggered many times when executing the "start" script.   not sure why. 

also, it writes a log line that tries to log something undefined:
```
> rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files

./lib/config.js -> dist/lib/config.js
./lib/services.js -> dist/lib/services.js
./lib/shipping-method.js -> dist/lib/shipping-method.js
./lib/update-credentials.js -> dist/lib/update-credentials.js
./lib/zones.js -> dist/lib/zones.js
this is the save value:undefined
```

Reply:
removed obsolete logline. ca1601b

some scripts like `import:shippingsmethods` will fail if the `build` script is not executed first. therefore those scripts run first the `build` script. 

the `npm start` is an accumulation of other scripts, in which some of them are calling the build.